shared/utilities/dateUtils/index.js

```diff
@@ -0,0 +1,22 @@
+import moment from 'moment-timezone';
+
+const timezone = 'Europe/London';
+
+export function formatDate(datetime, format = 'HH:mm') {
+  if (datetime) {
+    return moment(datetime.iso || datetime)
+      .tz(timezone)
+      .format(format);
+  }
+}
+
+export function isBefore(date1, date2) {
+  const firstDate = date1.iso || date1;
+  const secondDate = date2.iso || date2;
+
+  return moment(firstDate).tz(timezone).isBefore(moment(secondDate).tz(timezone));
```

Comment:
Perhaps we should make a function: convertTimeToLondonTime() or something better named